scripts/build-engine/src/index.ts

```diff
@@ -537,6 +540,11 @@ async function doBuild ({
         defaultHandler(warning);
     };
 
+    rollupPlugins.unshift(assetUrl({
+        engineRoot,
+        useWebGPU,
+    }));
+
```

Comment:
What does rollupPlugins do?

Reply:
copy wasm from `engine/native/external/emscripten` to the output engine lib `cococs-js/assets/`